db/db_tunables.c

```diff
@@ -710,6 +712,19 @@ static int deadlock_policy_override_update(void *context, void *value)
     return 0;
 }
 
+extern void clean_exit_sigwrap(int signum);
+
+static int update_clean_exit_on_sigterm(void *context, void *value) {
+    int val = *(int *)value;
+    if (val)
+        signal(SIGTERM, clean_exit_sigwrap);
+    else
+        signal(SIGTERM, SIG_DFL);
+    printf("turning SIGTERM handling %s\n", val ? "on" : "off");
```

Comment:
logmsg?

Reply:
Good catch, thanks.  I meant to take it out altogether.